core/scripts/i18n/collect-strings.js

```diff
@@ -814,7 +812,19 @@ function injectTraceEngineStrings() {
     const traceEngineStrings = JSON.parse(fs.readFileSync(traceEnginePath, 'utf-8'));
     const strings = JSON.parse(fs.readFileSync(file, 'utf-8'));
     for (const [key, value] of Object.entries(traceEngineStrings)) {
-      strings[`node_modules/@paulirish/trace_engine/${key.replace('.ts', '.js')}`] = value;
+      const lhKey = `node_modules/@paulirish/trace_engine/${key.replace('.ts', '.js')}`;
+
+      // TODO: why is this so malformed?
```

Comment:
Thought I recognized this haha. This looks like a translation bug that we fixed on the LH side but never upstreamed to DevTools: https://b.corp.google.com/issues/290679361

I guess it's a good thing we are using the TE strings now, so we can avoid fixing in one place but not the other. I will file a separate bug to resolve this again.

Reply:
Err, maybe this is a slightly different issue. The original DT string didn't contain any `\` at all which is different from LH...

Reply:
I filed b/399706272 about this, this workaround is fine for now